addons/pkg-npm/ftests/src/main/java/org/commonjava/indy/pkg/npm/content/NpmPathMappingTest.java

```diff
@@ -62,7 +64,7 @@ public void test()
         server.expect( server.formatUrl( REMOTE, PATH ), 200,
                        new ByteArrayInputStream( CONTENT_1.getBytes( "UTF-8" ) ) );
 
-        final RemoteRepository remote = new RemoteRepository( NPM_PKG_KEY, REMOTE, server.formatUrl( REMOTE ) );
+        final RemoteRepository remote = new RemoteRepository( NPM_PKG_KEY, REMOTE, server.formatUrl( REMOTE, SUBPATH ) );
```

Comment:
I'm not quite sure what this "subpath" here is working for. Seems it is breaking the ftest result.